milestone-3/frontend/src/components/home.tsx

```diff
@@ -1,11 +1,11 @@
 import "./home.css";
-import RecipePreview from "../recipePreview";
+import RecipePreview from "./recipePreview";
 import recipes from "../recipeData.json";
```

Comment:
instead of importing the recipe data for the second time here, you could have passed all the recipe data along as props to the Home element when you import it in app.tsx